source/accessibility.html.erb

```diff
@@ -84,14 +84,58 @@ title: Accessibility – GOV.UK Verify
         <p class="govuk-body">
           GDS is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
         </p>
+        <h2 class="govuk-heading-l">
+          Compliance status
+        </h2>
+        <p class="govuk-body">
+          This website is partially compliant with the <a href="https://www.w3.org/TR/WCAG21/" class="govuk-link">Web Content Accessibility Guidelines version 2.1</a> AA standard, due to the non-compliances listed below.
+        </p>
+        <h3 class="govuk-heading-l">
+          Non-accessible content
+        </h3>
         <p class="govuk-body">
-          This website is fully compliant with the <a href="https://www.w3.org/TR/WCAG21/" class="govuk-link">Web Content Accessibility Guidelines version 2.1</a> AA standard.
+        The content listed below is non-accessible for the following reasons.
+        </p>
+        <h4 class="govuk-heading-l">
+          Non-compliance with the accessibility regulations
+        </h4>
+        <p class="govuk-body">
+          The following content on Verify Status Page is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The non-compliances are listed below:
+        <ul class="govuk-list govuk-list--bullet">
+          <li>
+            an incorrect heading hierarchy which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)
+          </li>
+          <li>
+            missing level 1 headings (users may not be able to accurately determine the structure of content on the page)
+          </li>
+          <li>
+            our status incident page is missing the main heading and the status home page has an incorrect heading hierarchy. This fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships).
```

Comment:
We shouldn't include more than one sentence per bullet. You might be able to break these up with dashes instead, for example:

"our status incident page is missing the main heading and the status home page has an incorrect heading hierarchy - this fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)"

We don't put full stops at the end of bullets either.

In fact, there are a few style rules when it comes to using bullets! It might be worth double checking [the style guide entry on bullets](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps) to find out more.

Reply:
Thanks a lot Tom, these fixes have been implemented.